### DISCUSSION TITLE
why i can't remove F2A sign in request

### DISCUSSION THREAD

**Discussion by anhzzanh** [community/community#147747] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

I removed all F2A sign in request on all devices I have but github still request my F2A code, so I lost my account. How can i access my github account????

**Comment by ebndev** [comment]
Hi @anhzzanh,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
